app/pages/lab-pages-editor/components/TasksPage/components/EditStepDialog/types/DrawingTask.jsx

```diff
@@ -320,18 +327,33 @@ function DrawingTask({
         </span>
       </div>
       <div className="input-row">
-        <label
-          className="big spacing-bottom-S"
-          htmlFor={`task-${taskKey}-help`}
-        >
-          Help Text
-        </label>
-        <textarea
-          id={`task-${taskKey}-help`}
-          value={help}
-          onBlur={update}
-          onChange={(e) => { setHelp(e?.target?.value) }}
-        />
+        <div className="flex-row spacing-bottom-S">
+          <label
+            className="big"
+            htmlFor={`task-${taskKey}-help`}
```

Comment:
This requires the input to be on the page at all times. You can’t have a label without its corresponding input, so the show/hide pattern here is broken. It works visual but produces a broken DOM when the text input is removed. 

This needs to use a details/summary pattern, or the correct ARIA roles and state for a collapsible panel.

Reply:
Here's the WAI-ARIA pattern for building a [disclosure widget](https://www.w3.org/WAI/ARIA/apg/patterns/disclosure/), including roles, states and keyboard interactions.

Reply:
Alternatively, of course, you could use the [native HTML disclosure widget](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details).

Reply:
Scott O'Hara summarised [the pros and cons of using a native disclosure widget vs. rolling your own in JavaScript](https://www.scottohara.me/blog/2022/09/12/details-summary.html).